src/app.js

```diff
@@ -163,7 +163,9 @@ app.on('ready', () => {
     }
   }
 
-  app.dock.hide();
+  if (process.platform == 'darwin') {
```

Comment:
Could we please use strict equality checks? e.g. `process.platform === 'darwin'`